### PR TITLE
Build and test on Windows

### PR DESCRIPTION
**Pull request by badboy**

### CODE REVIEW

**Comment by badboy** [comment]
Oh, damn, expect a force-push as I messed up the rebase it seems

**Comment by badboy** [comment]
There we go.
AppVeyor succeeded on my account, if this is merged it should run on this repository as well (if not I have to figure out why)

File: examples/load_elf.rs [hunk 1]
```diff
@@ -118,10 +118,7 @@ fn main() {
...
-    vm.jit_compile();
-    unsafe {
-        let res = vm.prog_exec_jit(packet2);
-        println!("Packet #2, program returned: {:?} ({:#x})", res, res);
-        assert_eq!(res, 0);
-    }
+    let res = vm.prog_exec(packet2);
+    println!("Packet #2, program returned: {:?} ({:#x})", res, res);
+    assert_eq!(res, 0);
 }
```
Comment (qmonnet) [inline examples/load_elf.rs]: I'd like to keep some examples with JIT for Linux/MacOS. Do you think we could put this under a conditional guard and use the interpreter just for Windows?

File: examples/uptime.rs [hunk 2]
```diff
@@ -54,8 +54,7 @@ fn main() {
     vm.set_prog(prog2);
     vm.register_helper(helpers::BPF_KTIME_GETNS_IDX, helpers::bpf_time_getns);
 
-    vm.jit_compile();
-    let time = unsafe { vm.prog_exec_jit() };
+    let time = vm.prog_exec();
```
Comment (qmonnet) [inline examples/uptime.rs]: (same comment here)

**Comment by badboy** [comment]
Not pretty, but it should work.

**Comment by qmonnet** [comment]
Well, at least it shows how to use the JIT… Thanks! I'll merge your series.

**Pull request merged**